chapters/json-guidelines.adoc

```diff
@@ -53,7 +53,7 @@ of Internet Assigned Numbers Authority (IANA)) as `content-type` (or `accept`) h
 information. More specifically, for JSON payload you should use the standard media type
```

Comment:
At the beginning of this document it says:

> "These guidelines provides recommendations for defining JSON data at Zalando.
> JSON here refers to {RFC-7159}[RFC 7159] (which updates {RFC-4627}[RFC 4627]),
> the "application/json" media type and custom JSON media types defined for APIs.
> The guidelines clarifies some specific cases to allow Zalando JSON data to have
> an idiomatic form across teams and services."

Probably 'Zalando' should be replaced with 'IST'

Reply:
Thanks, I've updated it